tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,349 @@
+* A proposal requires only one vote to pass.
+
+```json
+{
+    "@type": "/cosmos.group.v1.ThresholdDecisionPolicy",
+    "threshold": "1",
+    "windows": {
+        "voting_period": "10m",
+        "min_execution_period": "0s"
+    }
+}
+```
+
+
+```sh
+simd tx group create-group-policy $ALICE $GROUP_ID "" policy.json
+```
+
+Verify your newly created group policy and save its address for future use:
+
+```sh
+simd query group group-policies-by-group $GROUP_ID
+export GROUP_POLICY_ADDRESS=$(simd query group group-policies-by-group $GROUP_ID --output json | jq -r '.group_policies[0].address')
+```
+
+## Create a proposal
+
+Now that you have a group with a few members and a group policy, you submit your first group proposal.
+Like for members management, you need to create a `proposal.json` file that contains the proposal.
+
+A proposal can contain any number of messages defined on the current blockchain.
+
+For this tutorial, you continue with your example of an association. The treasurer, Bob, who wants to send money to a third party to pay the bills, creates a `proposal.json`:
+
+
+```json
+{
+    "group_policy_address": "cosmos1..." , // the $GROUP_POLICY_ADDRESS
+    // array of proto-JSON-encoded sdk.Msgs
+    "messages": [
+        {
+            "@type": "/cosmos.bank.v1beta1.MsgSend",
+            "from_address": "cosmos1...", // the $GROUP_POLICY_ADDRESS
+            "to_address": "cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad", // a third-party
+            "amount": [
+                {
+                    "denom": "stake",
+                    "amount": "100"
+                }
+            ]
+        }
+    ],
+    "metadata": "costs utilities",
+    "proposers": [ "bobaddr" ] // $BOB
+}
+```
+
+This proposal, if passed, will send 100 `stake` to `cosmos1zyzu35rmctfd2fqnnytthheugqs96qxsne67ad` to pay the bills.
+The tokens will be sent from the decision policy address.
+
+<HighlightBox type="tip">
+
+The decision policy has no funds yet. You can fund it by sending a transaction with `simd tx bank send alice $GROUP_POLICY_ADDRESS 100stake`.
+
+</HighlightBox>
+
+
+```sh
+simd tx group submit-proposal proposal.json --from bob
+```
+
+## View and vote on proposals
+
+Check and verify the proposal of your football association:
+
+```sh
+simd query group proposals-by-group-policy $GROUP_POLICY_ADDRESS
+export PROPOSAL_ID=$(simd query group proposals-by-group-policy $GROUP_POLICY_ADDRESS --output json | jq -r '.proposals[0].id')
+```
+
+You can see that your proposal has been submitted.
+
+Next, have Alice and Bob vote _Yes_ on the proposal and verify that both their votes are tallied:
+
+```sh
+simd tx group vote $PROPOSAL_ID $ALICE VOTE_OPTION_YES "agree"
+simd tx group vote $PROPOSAL_ID $BOB VOTE_OPTION_YES "agree"
+simd query group tally-result $PROPOSAL_ID
+```
+
+Wait for the policy-prescribed 10 minutes, after which your proposal should have passed, as the weighted tally of _Yes_ votes is above the decision policy threshold:
```

Comment:
```suggestion
While you wait for the policy-prescribed 10 minutes, you can confirm that the final tally is still empty. After the 10 minutes have gone by your proposal should have passed, because the weighted tally of _Yes_ votes is above the decision policy threshold. Confirm this by looking at its `status`. It should be `PROPOSAL_STATUS_ACCEPTED`:
```